### DISCUSSION TITLE
Default value on Select field with multiple enabled

### DISCUSSION THREAD

**Discussion by Mark-van-Seeters-LW** [orchidsoftware/platform#2190] [Q&A]
When creating forms using Fields I often use the value() method to set a default value that is used when creating new model instances.
I know this is undocumented behaviour.
Now I am trying to use this on a select field that allows users to select multiple options.
The view for this field implies this is possible but it does nothing.
`@if (is_array($value) && in_array($key, $value, true)) selected`
$value is an empty array here

```
Select::make('companies')
    ->title('Companies')
    ->fromModel(Company::class, 'name', 'id')
    ->multiple()
    ->value(Company::query()->pluck('id')->toArray()), // this does not fill all companies by default
```

Has anybody managed to get this to work?

**Comment by yetki** [comment]
same same

**Comment by macit-emre** [comment]
You should try using ->empty() with your default value as parameters 
if value array is empty.
